src/components/App.tsx

```diff
@@ -98,7 +98,7 @@ export default function App() {
 
     // this state is pure React
     const [playing, setPlaying] = useState(false);
-    const [isLoadingPoints, setIsLoadingPoints] = useState(false);
```

Comment:
I start with false, to make sure the loading indicator is visible until the full TrackManager is loaded. It seems to work correctly for me on Safari/Macbook/iPad/iPhone Does this look okay to you?